apps/node-host/webpack.config.js

```diff
@@ -14,10 +14,14 @@ module.exports = composePlugins(withNx(), async (config) => {
     new UniversalFederationPlugin({
       isServer: true,
       name: 'node_host',
+      runtimePlugins: [require.resolve('./runtimePlugin.ts')],
       remotes: {
         node_local_remote:
           'commonjs ../../node-local-remote/dist/remoteEntry.js',
-        node_remote: 'node_remote@http://localhost:3002/remoteEntry.js',
+        // node_local_remote: '__webpack_require__.federation.instance.moduleCache.get("node_local_remote")',
+        node_remote:
+          '__webpack_require__.federation.instance.moduleCache.get("node_remote")@http://localhost:3002/remoteEntry.js',
```

Comment:
When using script global external modules in Webpack, it searches for containers in the internal module cache, as if it were a global variable.
![image](https://github.com/module-federation/universe/assets/25274700/13addf34-7485-44ed-9cb1-1c760ecdc57b)